Homework/test_form.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service as ChromeService
+from webdriver_manager.chrome import ChromeDriverManager
+from text_box_page import TextBoxPage
+
+@pytest.fixture
+def setup():
+    service = ChromeService(executable_path=ChromeDriverManager().install())
+    driver = webdriver.Chrome(service=service)
+    driver.get("https://demoqa.com/text-box")
```

Comment:
Instead of calling `driver.get` and `driver.maximize_window` methods directly in the setup fixture, consider moving them into the `TextBoxPage` class. This is part of the best practice to keep the test case ('test_submit_form') focus on the test steps, not on browser setup details.